core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -167,19 +165,19 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
                   put {
                     complete {
                       userServiceConstructor(ctx)
-                        .addUserToBillingProject(RawlsBillingProjectName(projectId),
-                                                 ProjectAccessUpdate(userEmail, ProjectRoles.withName(workbenchRole))
+                        .addUserToBillingProjectV2(RawlsBillingProjectName(projectId),
```

Comment:
Following the pattern of v2 methods handling BPM. Also this now lines up with the operationId values in the Swagger documentation.